features/animal.feature

```diff
@@ -1 +1,29 @@
-#Create your feature here
+Feature: Animal
+  In order to properly address all animals
+  As an animal
+  I want to provide my name, type and age
+  As an animal older than 3 years
+  I want to say I'm old
+
+
+  Scenario: Animal
+    Given an animal
+    When I ask about its "name"
+    Then I should get its "name"
+    When I ask about its "type"
+    Then I should get its "type"
+    When I ask about its "age"
+    Then I should get its "age"
```

Comment:
I feel like this scenario is a bit opaque.
I can't tell if the 'name' retrieved in the first Then is correct or just that we could retrieve one.

How about something like

    Scenario: getting the animal's name
        Given we have a 2 year old dog named 'Fido'
        Then the animal's name should be 'Fido'
        And  the animal's type should be 'dog'
        And  the animal's age should be 2